### ISSUE TITLE
Set window.$ = blingblingjs in UMD build

### ISSUE THREAD

**Issue by chriscalo** [argyleink/blingblingjs#7]
Excited to try out blingblingjs for quick prototyping without a build step. I dropped it into an HTML file via unpkg.com, and I was surprised that `window.$` was undefined. Turns out that `window.blingblingjs` is what gets defined, so to use `$` as seems to be the convention from the README, I have to do the following:

``` html
<!DOCTYPE html>
<script src="//unpkg.com/blingblingjs@2.0.2/dist/index.js"></script>
<script>
  var $ = blingblingjs;
</script>
```

Proposal:
1. Continue setting `window.blingblingjs`
2. If `window.$` is undefined, set it to point to blingblingjs

**Comment by argyleink** [comment]
oo nice, i wonder if there's an easy rollup config for this. like setting alias's for non module scenarios. looks like the default is to use the package name as the window global. cool man, i'll check it out! good call.

**Comment by argyleink** [comment]
easy fix once i investigated. all tests are passing in the PR, i'll merge and release 👍

**Comment by argyleink** [comment]
fixed in v2.1.0

**Comment by chriscalo** [comment]
Right on, thanks for the quick fix! 🤘 Now I want to 👀 how you did it…
